product/dialogs/miq_dialogs/miq_provision_microsoft_dialogs_template.yaml

```diff
@@ -488,6 +488,9 @@
           :data_type: :boolean
         :vm_memory:
           :values:
+            "32768": "32768"
+            "16384": "16384"
+            "12288": "12288"
```

Comment:
@bronaghs Do these static VM memory values look ok for the default SCVMM provisioning dialogs?
Note: Users can modify to add/remove values.